test/test_client.py

```diff
@@ -1003,6 +1003,20 @@ def callback(FUsrPtr, JobOp, response):
         self.assertEqual(expected, self.client.ct_read(0, 1))
         self.assertEqual(1, self.callback_counter)
 
+    def test_read_write_integer_area(self):
+        amount = 1
+        start = 1
+        valid_area = 28
+        dbnumber = 0
+        data = bytearray(b'\x13\x35')
+        self.client.write_area(valid_area, dbnumber, start, data)
+        res = self.client.read_area(valid_area, dbnumber, start, amount)
+        self.assertEqual(data, bytearray(res))
+
+        invalid_area = 777
+        self.assertRaises(Snap7InvalidArea, self.client.read_area, invalid_area, dbnumber, start, amount)
```

Comment:
do we also test valid ints and Area as arguments for read_area?

Reply:
`test_write_area` and `test_read_area`  use the `Areas` type. `test_read_write_integer_area`  also checks a valid integer (line 1009).

Reply:
ah yes, sorry, i could have checked but was on my mobile phone.